src/com/google/javascript/jscomp/PolymerClassDefinition.java

```diff
@@ -33,6 +33,8 @@
  * class.
  */
 final class PolymerClassDefinition {
+  /** Either 1 or 2 - the Polymer version of the definition */
+  final int version;
```

Comment:
@MatrixFrog is recommending a new flag name, like `--polymer_version=VERSION` and for a while `--polymer_pass` will be an alias for `--polymer_version=1`